src/main/resources/export/java/RobotTemplate-autonomous.java

```diff
@@ -1,5 +1,12 @@
 #set($command = $robot.getProperty("Autonomous Command").getValue())
-#set($params = $robot.getProperty("Autonomous command parameters").getValue())
 
-#if( $command != "None" )        autonomousCommand = #command_instantiation( $command $params );
+#foreach( $component in $components )
+#if ($component.getBase().getType() == "Command"
+     && $component.getProperty("Autonomous Selection").getValue())
+        chooser.addObject("$component.getName()", new #class($component.getName())());
```

Comment:
This doesn't support parameterized commands. We should add one autonomous choice per preset. Parameterized commands with no presets shouldn't be added

For example, if I have a command with parameters `x` and `y`, and various presets for those parameters, each preset should appear in the chooser.